### ISSUE TITLE
SS5 and ACIA

### ISSUE THREAD

**Issue by THE-MrMicro** [GideonZ/1541ultimate#221]
Ok, had this issue using SS5 (PAL/NTSC) and ACIA enabled that is now resolved and here's the reasoning and final (only) options.

1. Stay on $DE00 and have SS5 disabled. Use your favorite Terminal to call some BBS's. (REU on or off, your discretion).

OR

2. Switch to $DF00 and KEEP SS5 enabled, BUT DISABLE REU since it conflicts with $DF00 and use NovaTerm 9.1/9.6 or CCGMS Ultimate and set to Swiftlink/D7.

**It's NOT a limitation or error in the U64 or 1541U2+**, it's simply a software/hardware limitation because of these addresses. This would happen on real hardware too. The SS5 software uses address $DE00 and the REU occupies $DF00 and the only terminals supporting ACIA (Novaterm & CCGMS) look for the 6551 at those addresses.

To me, the SS5 is useless when calling BBS's and WOULD NOT supersede having an REU active if for nothing else, BUFFER loading, so I'll be sticking with $DE00 and simply disabling SS5.

Of course, none of this matters if YOU'RE NOT using SS5 and ACIA anyway, but if you are, well, now ya know.
_________________
I've seen the future.... and I want to go back!

C-NET BBS: micromansion.servebbs.net:6400

**Comment by p2mate** [comment]
I noticed SS5 hangs at startup if SID auto address mirroring is turned off. No idea why.

**Comment by THE-MrMicro** [comment]
I haven’t found that and I JUST turned that off 2-3 days ago when installing the ARM2SIDs in my U64.

of course it did hang when I used 3 SID config because it occupies I/O 1 (DE00), same problem as ACIA.

**Comment by p2mate** [comment]
Interesting. I don't have any 'external' SIDs, only the UltraSIDs which are part of the u64 FPGA bitstream.

**Comment by THE-MrMicro** [comment]
> 
> 
> I noticed SS5 hangs at startup if SID auto address mirroring is turned off. No idea why.

Actually, I just went back and confirmed and I DID NOT have to turn off mirroring, I turned off Auto Addressing. I DID just test your concern and turned off address mirroring and turned on SS5 and I COULD NOT even get the U64 to boot, well at least to the SS5 menu screen. All I got was a black screen even after a complete power down and on. 

I was able to press the power button to get in to the menu and enable mirroring again, so I'm not sure what's going on there.
